.github/workflows/cross_musl.yml

```diff
@@ -30,5 +30,5 @@ jobs:
 
       - name: Tests
         run: |
-          xmake l ./scripts/test.lua -D -p cross --toolchain=muslcc --sdk=`pwd`/${{ matrix.cross }}-cross
+          xmake l ./scripts/test.lua -vD -p cross --toolchain=muslcc --sdk=`pwd`/${{ matrix.cross }}-cross
```

Comment:
it works now, please remove `-v`

Reply:
I was wondering why this wasn't a default on xmake-repo CI, since it brings a lot of info

Reply:
Some packages output a lot of logs, making compilation very slow and difficult to locate critical error messages.

There is also a limit to the size of the logs displayed on the ci, resulting in me having to download the full logs to find the error messages.